engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ConstraintValidatorManager.java

```diff
@@ -115,13 +115,13 @@ public ConstraintValidatorManager(ConstraintValidatorFactory constraintValidator
 			}
 		}
 
-		Class<? extends ConstraintValidator<?, ?>> validatorClass = findMatchingValidatorClass(
+		ConstraintValidatorDescriptor<A> validatorTypeDescriptor = findMatchingValidatorClass(
```

Comment:
`validatorDescriptor` would be nice to be consistent with the naming a few lines below.